### DISCUSSION TITLE
Page vanishes when renamed

### DISCUSSION THREAD

**Discussion by AllTheUserNamesAreGone** [filamentphp/filament#12060] [Help] [answered]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hi,

I have created some pages using:

php artisan make:filament-page Settings

I have had to rename the pages (via VS Code or OS) later and they have now vanished from the side nav. I was expecting them to be detected as they were when I created them but they are not and can't see where/how they were added in the first place. How do you tell filament to look for a different page name after a page has been renamed?

Thanks

**Comment by leandrocfe** [comment, marked as answer]
Did you rename the file and the class name?

if you create a page `php artisan make:filament-page Settings` and you want to rename this to Configuration, you should rename the file and the class
`app/Filament/Pages/Configuration.php`
```php
class Configuration extends Page
{
}
```

**Comment by AllTheUserNamesAreGone** [comment]
No I did not. It is now working. Thank you so very very much, I wasted so much frustrating time Googling/Binging "Filament Page Rename etc" trying to figure out where it "registered" the page and it was staring me right in the face. Thank you.
